mne/report.py

```diff
@@ -814,6 +824,25 @@ def __init__(self, info_fname=None, subjects_dir=None,
 
         self._init_render()  # Initialize the renderer
 
+    def __repr__(self):
+        """Print useful info about report."""
+        s = '<Report | %d items' % len(self.fnames)
+        if self.title is not None:
+            s += ' | %s' % self.title
+        fnames = [_get_fname(f) for f in self.fnames]
+        if len(self.fnames) > 4:
+            s += '\n%s' % '\n'.join(fnames[:2])
+            s += '\n ...\n'
+            s += '\n'.join(fnames[-2:])
+        elif len(self.fnames) > 0:
+            s += '\n%s' % '\n'.join(fnames)
+        s += '\n>'
+        return s
```

Comment:
I added a couple more lines in the tests and verified that I was hitting all the branches